src/home-page/home-page.component.scss

```diff
@@ -54,7 +55,7 @@
     height: 10vh;
     display: flex;
     text-decoration: none;
-    color:#212529;
+    color: #212529;
```

Comment:
If you have some general layout plan in mind, consider moving colors to separate file, eg. `colors.scss`. Then you can import this file here via `@import './colors.scss';`